.github/workflows/maven_release.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           passphrase: ${{ secrets.GPG_PASSPHRASE }}
       - name: 'Publish to Maven'
         run: |
-          ./gradlew --no-parallel --no-daemon publishAllPublicationsToOSSRHRepository -Psigning.keyId=${{secrets.GPG_KEY_ID}} -Psigning.password=${{secrets.GPG_PASSPHRASE}} -Psigning.secretKeyRingFile=$(echo ~/.gradle/secring.gpg)
+          ./gradlew --no-parallel --no-daemon publishAllPublicationsToMavenRepository -Prelease=true -Psigning.keyId=${{secrets.GPG_KEY_ID}} -Psigning.password=${{secrets.GPG_PASSPHRASE}} -Psigning.secretKeyRingFile=$(echo ~/.gradle/secring.gpg)
```

Comment:
There's a new parameter "jacocoEnabled". I think to releases we need to explicitly disable jacoco.